packages/@vue/cli-upgrade/get-upgradable-version.js

```diff
@@ -0,0 +1,38 @@
+const execa = require('execa')
+
+function getMaxSatisfying (packageName, range) {
+  let version = JSON.parse(
+    execa.shellSync(`npm view ${packageName}@${range} version --json`).stdout
```

Comment:
What if `npm` is not installed?

Reply:
But how is it possible?

Reply:
...but then you can't really run a vue-cli project, so you have bigger problems than `vue upgrade` not working.